app/templates/registercontainer.html

```diff
@@ -40,7 +40,7 @@ <h4 class="modal-title">Register Tool</h4>
           ng-if="refreshingContainer">
         <span class="glyphicon glyphicon-refresh glyphicon-refresh-animate">
         </span>
-        Please wait, retrieving Quay.io and/or Docker Hub container...
+        Please wait, retrieving Quay.io and/or Docker Hub tool...
```

Comment:
Strictly, I think a tool is either from Quay.io or Docker Hub, but cannot be "and"